azure-pipelines.yml

```diff
@@ -162,6 +162,56 @@ jobs:
               /p:TeamName=DotNetCore
               /p:OfficialBuildId=$(BUILD.BUILDNUMBER)
             displayName: Sign packages
+    - job: Run_SDL
+      displayName: Run SDL tool
+      variables:
+        - group: SDL_Eng_KeyVault
+      dependsOn:
+        - Windows_NT
+        - Linux
+        - Validate_Helix
+        - Validate_Signing
+        - Asset_Registry_Publish
+      steps:
+        - checkout: self
+          clean: true
+        - powershell: eng\common\build.ps1
+            -configuration Release
+            -restore
+            -prepareMachine
+            -ci
+            /p:OfficialBuildId=$(BUILD.BUILDNUMBER)
+          displayName: Windows Build / Publish
+        - task: NuGetToolInstaller@1
+          displayName: 'Install NuGet.exe'
+        - task: NuGetCommand@2
+          displayName: 'Install Guardian'
+          inputs:
+            restoreSolution: $(Build.SourcesDirectory)\eng\common\sdl\packages.config
+            feedsToUse: config
+            nugetConfigPath: $(Build.SourcesDirectory)\eng\common\sdl\NuGet.config
+            externalFeedCredentials: GuardianConnect
```

Comment:
Where is this defined?

Reply:
On the dnceng AzDO instance, shared by all repos running there.

Reply:
I'm assuming it's only available to internal projects?